fix-dnsmasq-options

sgoveas opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- Adds scheduled bare-metal IPv4 test jobs for disconnected and day-2 virtual-media scenarios across OpenShift 4.21, 4.22, and 5.0.
- Adds the disconnected bare-metal IPI virtual-media chain and workflow, including metadata and ownership links.
- Sets a fixed CI release image for bare-metal IPI installation.
- Allows incompatible multi-architecture worker configurations to exit successfully.
- Extends cluster duration and changes the wait step image.
- Disables post-install QE test steps after cluster health checks and disables agent log gathering.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse 	periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

aleskandro commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/remove-lifecycle stale

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse 	periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7 periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7 periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7 periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7 periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f28 periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f28 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-upi-ovn-ipv4-disconnected-proxy-arm-f14 periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-4.18-upgrade-from-stable-4.17-baremetal-upi-ovn-dual-stack-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-4.17-upgrade-from-stable-4.16-baremetal-ipi-ovn-ipv4-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28

sgoveas commented:
/reopen

sgoveas commented:
/remove-lifecycle rotten

sgoveas commented:
/open

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/remove-lifecycle stale

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f28 periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-4.18-upgrade-from-stable-4.17-baremetal-ipi-ovn-ipv4-external-lb-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-amd64-nightly-baremetal-upi-ovn-ipv4-f14 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-baremetal-upi-ovn-ipv4-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-amd-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-arm-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-amd-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28 periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-amd-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-amd-disruptive-f14 periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-amd-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-amd-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-day2-amd-disruptive-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-arm-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28 periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-disconnected-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-external-lb-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-dualstack-arm-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-baremetal-ipi-ovn-dualstack-arm-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-dualstack-amd-vmedia-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-dualstack-amd-vmedia-f7 periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-multi-arch-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14 periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f7 periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14 periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-amd64-nightly-baremetal-ipi-ovn-vmedia-disk-encryption-f7 periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14 periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7 periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-basecap-none-arm-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-arm-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14

sgoveas committed:
skip test, use baremetal-qe-base for wait

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.21-amd64-nightly-baremetal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-amd-f28 periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-multi-arch-day2-amd-f28

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-capi-tp-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-capi-tp-amd-f14

sgoveas committed:
add amd64 disconnected IPI

sgoveas committed:
add bmaas jobs 4.21 and 4.22
use 2 workers for bmaas
add disconnected vmedia 4.22 job

periodic jobs not created

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-amd64-nightly-baremetal-ipi-ovn-ipv4-disconnected-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-vmedia-amd-f14 periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-ipi-ovn-ipv4-amd-f14

sgoveas committed:
add the jobs bmaas and arm diconnected

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-metal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-metal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-metal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-metal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-metal-ipi-ovn-ipv4-multi-arch-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-metal-ipi-ovn-ipv4-multi-arch-day2-amd-f14

sgoveas committed:
add periodics

sgoveas committed:
use 1 additional for 4.22

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-amd64-nightly-metal-upi-ovn-ipv4-f7

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-metal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14

sgoveas commented:
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.22-multi-nightly-metal-ipi-ovn-ipv4-vmedia-bmaas-day2-amd-f14
